packages/concerto-core/lib/introspect/mapkeytype.js

```diff
@@ -162,6 +162,14 @@ class MapKeyType extends Decorated {
     isValue() {
         return false;
     }
+
+    /**
+     * Return the namespace of this map key.
+     * @return {string} namespace - a namespace.
```

Comment:
nit, I think by now this is very clear that this is a `namespace` :D

Reply:
Its a convention that I've stuck to from similar method in other classes. Moreover JSDoc refers to these comments to build API reference docs.